hoot-services/src/main/java/hoot/services/controllers/conflation/ConflateCommand.java

```diff
@@ -182,19 +175,22 @@ else if (conflationCommand != null && conflationCommand.contains("differential")
             });
         }
 
+        //Attribute Reference Layer is not default
+        if ("2".equals(params.getReferenceLayer())) {
+            options.add("tag.merger.default=hoot::OverwriteTag1Merger");
+        }
```

Comment:
This got whacked back in March, but implements setting the Attribute Reference Layer from the UI
We now have a test for this.